### PR TITLE
Fix code coverage comparison

### PR DESCRIPTION
**Pull request by vsalvino**
Build keeps failing when trying to download previous code coverage from master branch.

Updating YAML keywords which seem to have magically changed in Azure without any notice or version change. Also setting code coverage comparison to pass with warnings if any errors occur.

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -96,28 +96,35 @@ stages:
...
   - job: codecov
-    displayName: Code Coverage
+    displayName: 'Code Coverage'
+    continueOnError: true
     pool:
       vmImage: 'ubuntu-latest'
 
     steps:
+    - pwsh: write-output "project $Env:project, pipeline $Env:pipeline"
+      displayName: 'Helpppppp'
```
Comment (corysutyak) [inline azure-pipelines.yml]: Are the multiple p's intentional?

Comment (vsalvino) [reply azure-pipelines.yml]: I'm just shooting in the dark at this point trying to figure out what's going on... hence the work in progress label.

**Comment by vsalvino** [comment]
My theory is that this is failing because it is not downloading an artifact since every available master build is "fail" status.

At one point all of the master builds got deleted due to a 30 day retention limit, and we had not merged to master in over 30 days at one point. Every build after that technically failed due to missing the previous master build.

Setting the code coverage to pass with warnings if it is not able to compare it to previous builds. Will try running this in master twice to see if that fixes the problem and the second build is able to download from the first build after this change.

**Comment by vsalvino** [comment]
Cancelling in favor of #261

**Pull request closed without merging**